src/util/getExperienceName.ts

```diff
@@ -0,0 +1,41 @@
+import { BlobServiceClient, ContainerClient } from "@azure/storage-blob";
+
```

Comment:
I think it's better to change the name of the script or move the code related to getting files in the blob to a separate code file

Reply:
Thanks for the suggestion!